.github/copilot-instructions.md

```
@@ -559,11 +561,100 @@ The `Qwiq.Package.Tests` project validates NuGet package contents using Verify.
+| ID  | Type       | Title                                      | Purpose                    |
+| --- | ---------- | ------------------------------------------ | -------------------------- |
+| 1   | Bug        | Integration Test                           | Basic work item tests      |
+| 2   | Task       | Child Task for Integration Tests           | Child of ID 3 (hierarchy)  |
+| 3   | User Story | Parent Story for Integration Tests         | Parent for hierarchy tests |
+| 4   | Bug        | Bug for Mapper Integration Tests           | Mapper tests               |
+| 5   | Bug        | Work Item with Links for Integration Tests | Work item with links       |
+| 6   | Task       | Child Task 2 for Hierarchy                 | Second child of ID 3       |
+
+**Hierarchy Structure:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)